<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfinityAscToBasDialog</class>
 <widget class="QDialog" name="InfinityAscToBasDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Infinty ASC to Comp3D BAS conversion</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditAscFile">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Press &quot;Help&quot; to see an example</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButtonAscFile">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Conversion result:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextBrowser" name="textBrowserResult"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonHelp">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>Press &quot;Help&quot; to see an example</string>
     </property>
     <property name="text">
      <string>In: ASC File</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>InfinityAscToBasDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
